app/[locale]/[[...path]]/page.tsx

```diff
@@ -67,7 +67,7 @@ const getPage: FC<DynamicParams> = async ({ params }) => {
     // Forces the current locale to be the Default Locale
     unstable_setRequestLocale(defaultLocale.code);
 
-    return notFound();
+    return redirect(`/${defaultLocale.code}`);
```

Comment:
But wont that redirect all routes to default locale? I think the intent is to redirect to a page that exists in /en whenever it exists.